frontend/src/components/common/comments-list/comments-list.tsx

```diff
@@ -3,15 +3,17 @@ import { Comment } from 'common/types/types';
 import styles from './styles.module.scss';
 
 type Props = {
+  hasRecord?: boolean;
```

Comment:
```suggestion
  hasRecord: boolean;
```
Why do you use 'undefined' type here?

Reply:
it's okay because we can have cases when we don't have an record

Reply:
Yes, in live record, we have comments but don't have record